src/packages/@ncigdc/modern_components/SCRNASeq/toolbar/DownloadButton.js

```diff
@@ -1,85 +1,53 @@
-import React, { Component } from 'react';
-
 import DropDown from '@ncigdc/uikit/Dropdown';
 import DropdownItem from '@ncigdc/uikit/DropdownItem';
 
 import ToolbarButton from './ToolbarButton';
 
-const downloadOptions = {
-  downloadAnalysis: [
```

Comment:
downloadAnalysis moved to its own component because appearance & functionality diverged